pkg/apis/velero/v1/server_status_request.go

```diff
@@ -56,6 +58,9 @@ type ServerStatusRequestStatus struct {
 
 	// ServerVersion is the Velero server version.
 	ServerVersion string `json:"serverVersion"`
+
+	// Plugins list information about the plugins running on the Velero server
+	Plugins [][]string `json:"plugins"`
```

Comment:
Instead of having nested slices here, I would define a new struct to capture the plugin info, something along the lines of:

```go
type PluginInfo struct {
    Name string // TODO json tags
    Kind string // TODO json tags
}
```

And then it becomes:

```go
Plugins []PluginInfo
```